docker-bake.hcl

```diff
@@ -93,6 +93,7 @@ target "lint" {
       { name = "labs", tags = "dfrunsecurity", target = "golangci-lint" },
       { name = "nydus", tags = "nydus", target = "golangci-lint" },
       { name = "yaml", tags = "", target = "yamllint" },
+      { name = "proto", tags = "", target = "protolint" },
```

Comment:
Can we also add a make target for this (or make this part of one of the existing ones)?

Reply:
`make lint` already includes this - it runs all the targets here in parallel.